test/integration/integration_test.cc

```diff
@@ -220,7 +220,10 @@ TEST_P(IntegrationTest, WebSocketConnectionUpstreamDisconnect) {
 
 class BindIntegrationTest : public IntegrationTest {
 public:
-  void SetUp() override {
+  // Delay base class SetUp until initialize().
```

Comment:
Why is this needed?

Reply:
To allow the bind-fail test to override the address before set-up.

I'm fairly convinced that in the long run we should ditch SetUp for Initialize() (to allow common config manipulation) but I figured on doing that in one of the PRs for simplifying config set-up

Reply:
Sure. Maybe just add an explanation of why the delay then in the comment.